.github/pull_request_template.md

```diff
@@ -15,13 +15,15 @@
 - [ ] My code follows the code style of this project.
 - [ ] My change requires a change to the documentation.
   - [ ] I have updated the documentation accordingly.
+  - [ ] I have recorded my changes in the changelog for the upcoming release 
```

Comment:
Do we want to recommend when a change is necessary for the changelog? I do think putting this as a documentation requirement is good, but I'm trying to think about minimal changes that might not be necessary to put in the changelog. Maybe it's sufficient to trust our contributors to make the right call. What if we said something like: 

```suggestion
  - [ ] If I have made a user-facing change, I have recorded my changes in the changelog for the upcoming release 
```

I'm not sure. What do you think @yardasol ?

Reply:
Ooh I like that better!